can/interfaces/vector/vxlapi.py

```diff
@@ -0,0 +1,859 @@
+# -*- coding: utf-8 -*-
+"""
+Ctypes wrapper module for Vector CAN Interface on win32/win64 systems
+Authors: Julien Grave <grave.jul@gmail.com>
+"""
+# Import Standard Python Modules
+# ==============================
+import ctypes
+import logging
+import platform
+import sys
+from can import CanError
+
+# Define Module Logger
+# ====================
+LOG = logging.getLogger(__name__)
+
+# Vector XL API Definitions
+# =========================
+# Load Windows DLL
+_xlapi_dll = None
+if sys.platform == 'win32':
+    architecture = platform.architecture()[0]
+    try:
+        if architecture == '64bit':
+            _xlapi_dll = ctypes.windll.LoadLibrary('vxlapi64')
+        else:
+            _xlapi_dll = ctypes.windll.LoadLibrary('vxlapi')
+    except OSError:
+        if architecture == '64bit':
+            LOG.error('Cannot load Dynamic XL Driver Library (vxlapi64.dll)')
+        else:
+            LOG.error('Cannot load Dynamic XL Driver Library (vxlapi.dll)')
+else:
+    LOG.warning('This library is only availabe for Windows')
+
+# Bus types
```

Comment:
True. Most of the code in this module is not in use so I was thinking of stripping it down to make it easier to find the stuff that matters.